README.md

```diff
@@ -144,7 +144,7 @@ jobs:
     steps:
     - uses: actions/checkout@v3
     - name: Install Nix
-      uses: DeterminateSystems/nix-installer@main
+      uses: DeterminateSystems/nix-installer-action
```

Comment:
I think it's required to specify a version / ref here...? Maybe?

Reply:
I don't think it is! :grimacing:

Reply:
:scream: